### PR TITLE
Mobile menu Pull Request

### PR DESCRIPTION
**Pull request by fabianofrank**
Implementation of hamburger menu at the mobile version of the portfolio

### CODE REVIEW

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @fabianofrank, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing eslint. For this particular failing linter, you can use eslint inline comment to make it pass, but only for this failing linter since you're using the function in another file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -6,20 +6,49 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <link rel="stylesheet" href="queries.css">
+  <script src="menu.js"></script>
```
Comment (carlos-ssh) [inline index.html:9]: - Fix your icons in your ``menu`` once that you click on your ``hamburger menu``, the ``X`` keeps displayed.

You can work displaying and hidding the icons; using the``class`` icons in a example.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @fabianofrank , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._


changesrequested




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rotimiazeez — APPROVED** [review]
# Hi Team 🎉, 

### Your project is complete👏👏! You have implemented the required changes🕺... it's time to merge it :shipit: 🚀
Congratulations! 🎉 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**